openedx_external_enrollments/tests/test_signal_receivers.py

```diff
@@ -0,0 +1,74 @@
+"""Tests SalesforceEnrollment class file."""
+from django.test import TestCase
+from mock import Mock, patch
+
+from openedx_external_enrollments.signal_receivers import update_external_enrollment, delete_external_enrollment
+
+
+class UpdateExternalEnrollmentTest(TestCase):
+    """Test class for update_external_enrollment method."""
+
+    @patch('openedx_external_enrollments.signal_receivers.get_course_by_id')
+    def test_update_external_enrollments(self, get_course_by_id_mock):
+        """Testing update_external_enrollments method."""
+        instance = Mock()
+        instance.is_active = False
+        instance.mode = 'test-mode'
+        instance.user.email = 'test-email'
+        instance.course.id = 'test-course-id'
+        get_course_by_id_mock.return_value = 'test-course'
+        data = {
+            'user_email': instance.user.email,
+            'course_mode': instance.mode,
+            'is_active': False,
+        }
+
+        with patch('openedx_external_enrollments.signal_receivers.execute_external_enrollment') as execute_mock:
+            update_external_enrollment('fake-sender', True, instance)
+
+            assert not get_course_by_id_mock.called, 'get_course_by_id was called and should not have been'
```

Comment:
Isn't this the same message self.assert_not_called() displays? @andrey-canon 
What 's the point of putting the same message here?

Reply:
Thi is not `assert_not_called`, this is a normal assert, I'm not using `assert_not_called` because edx-platform uses the version 1.0.1 of mock, I think we can update  mock since this is a test scenario.